tiny_ocl.h

```diff
@@ -37,6 +37,44 @@ THE SOFTWARE.
 #include "cl.h"
 #include <vector>
 
+// aligned memory allocation
```

Comment:
One thing I was wondering: What happens when tiny_bvh.h and tiny_ocl.h would use custom allocation functions with the same name, but in a different namespace? That would simply work right? I used 'unusual struct names' in tiny_bvh to avoid any name clashes, like bvhvec3, but perhaps that wasn't needed in the first place? OTH, if someone would use 'using namespace tinybvh', things would get messy if bvhvec3 would simply be named vec3 I guess. Requesting your advice.

Reply:
Yes I would work if they are in different namespaces: you just have to use the fully qualified name (tinybvh::default_aligned_alloc, tinyocl::default_aligned_alloc).

I would say in general it is not a good practice to use `using namespace tinybvh` with a wide scope: either use the fully qualified name or restrict the `using` to the scope of a function for example (but this is really just my opinion).

OTH you could also remove the namespace altogether and use a `tinybvh_` prefix everywhere (tinybvh_vec3 for example).
This would be the C-style approach.